plugin/output/elasticsearch/elasticsearch.go

```diff
@@ -163,6 +169,10 @@ func (p *Plugin) Start(config pipeline.AnyConfig, params *pipeline.OutputPluginP
 		p.config.IndexValues = append(p.config.IndexValues, "@time")
 	}
 
+	if p.config.BatchOpType != "index" && p.config.BatchOpType != "create" {
```

Comment:
It can be automatically validated using `options` in the config description. Like here: https://github.com/ozontech/file.d/blob/master/plugin/output/gelf/gelf.go#L99